src/Compilers/CSharp/Portable/Symbols/Source/SourceUserDefinedOperatorSymbolBase.cs

```diff
@@ -247,10 +264,16 @@ protected sealed override void ExtensionMethodChecks(BindingDiagnosticBag diagno
 
         protected sealed override MethodSymbol FindExplicitlyImplementedMethod(BindingDiagnosticBag diagnostics)
         {
+            if (_explicitInterfaceType is object)
+            {
+                var syntax = (OperatorDeclarationSyntax)syntaxReferenceOpt.GetSyntax();
+                return this.FindExplicitlyImplementedMethod(isOperator: true, _explicitInterfaceType, OperatorFacts.OperatorNameFromDeclaration(syntax), syntax.ExplicitInterfaceSpecifier, diagnostics);
+            }
+
             return null;
         }
 
-        protected sealed override TypeSymbol ExplicitInterfaceType => null;
+        protected sealed override TypeSymbol ExplicitInterfaceType => _explicitInterfaceType;
```

Comment:
>_explicitInterfaceType [](http://example.com/codeflow?start=70&length=22)

Why use a dedicated backing field for this, rather than just an autoprop?

Reply:
> Why use a dedicated backing field for this, rather than just an autoprop?

Personal preference. Don't like shortcuts that hide things.